packages/analytics-js/__mocks__/msw.handlers.ts

```diff
@@ -1,5 +1,5 @@
 import { rest } from 'msw';
-import { dummyDataplaneHost } from './fixtures';
+import { dummyDataplaneHost, dummySourceConfigResponse } from './fixtures';
 
 // TODO: Why on data plane we allow-origin the domain but in sourceConfig is wildcard?
```

Comment:
@bardisg This should not be the case. Do you see otherwise?

Reply:
I believe that serverside shouls always allow as cross orgin the refereer URL. Having said this the sourceConfig endpoint response comes from CloudFront and by using different response headers as cache criteria would probabaly blow up cost. 

This is something to consider afteer we do the v3 MVP and if server can fix it we can potentally start using fetch instead of legacy XHR (as fetch does not allow CORS requests with credentials adn wildcard), thus the todo in the comments